content/SUMMARY.md

```diff
@@ -5,9 +5,10 @@
   * [How to contribute](meta/contributing.md)
   * [FAQ](meta/faq.md)
 * [WordPress Blog](meta/wordpress_blog.md)
-* [Environment](meta/environment.md)
-  * [Mac](meta/mac.md)
-  * [Linux](meta/linux.md)
+* [Environment](environment/index.md)
```

Comment:
FYI:  this will rename http://wpveda.com/meta/ to http://wpveda.com/environment/ — which is good, so please move ahead. 👍